policyengine_us/parameters/gov/states/or/tax/income/deductions/standard/claimable_as_dependent/min.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   unit: currency-USD
   period: year
   label: Oregon minimum deduction if claimable as dependent
+  uprating: gov.irs.uprating  
```

Comment:
rounded -> all parameters that seem to be adjusted to a round number should be checked for rounding